apps/docs/src/app/core/component-docs/busy-indicator/examples/busy-indicator-extended-example.component.html

```diff
@@ -1,4 +1,4 @@
-<button fd-button label="Open from Template" (click)="openFromTemplate(template)"></button>
+<button style="margin-right: 8px" fd-button label="Open from Template" (click)="openFromTemplate(template)"></button>
```

Comment:
There shouldn't be a margin at all:
```
<button fd-button label="Open from Template" (click)="openFromTemplate(template)"></button>
```

Reply:
I think we need for that example to have space between the buttons.
<img width="235" alt="Screen Shot 2022-08-19 at 2 15 09 PM" src="https://user-images.githubusercontent.com/65063487/185681809-f297b8a9-c91c-4bfd-af08-6909ce59291b.png">